Projects/Feature/History/Sources/HistoryStore.swift

```diff
@@ -95,12 +119,12 @@ public struct HistoryStore {
         return .none
         
       case .didTapDeleteCancel:
-        state.showDeleteAlert = false
+        state.modal = nil
         state.selected = nil
         return .none
```

Comment:
아래 `didTapEditTitleCancel`, `didTapDismiss`에서 중복된 코드가 반복돼요!

별도의 로직 추가가 없을 것 같다면, 아래처럼 변경하고
```suggestion
      case .didTapDeleteCancel, .didTapEditTitleCancel, .didTapDismiss:
        state.modal = nil
        state.selected = nil
        return .none
```

아니라면 아래와 같이 별도의 action을 하나 추가해서 처리하는게 좋아보여요!
```suggestion
      case .didTapDeleteCancel:
        return .send(.dismissModal)
```